src/I18n/DateFormatTrait.php

```diff
@@ -30,8 +30,17 @@
      * The default locale to be used for displaying formatted date strings.
      *
      * @var string
+     * @deprecated Use self::setDefaultLocale($locale) and self::getDefaultLocale() instead.
```

Comment:
Here I think this should stay in. It is bad to not know early up that you are doing things wrong. Also it makes sure that once 4.x hits it is becoming a protected property and isn't forgotten.